### ISSUE TITLE
Labels inside textarea don't use the global background color

### ISSUE THREAD

**Issue by zandaqo** [material-components/material-components-web#3292]
## Bugs

A floating label inside a textarea field uses white color as its backgroud rather than that of the textarea which uses the --mdc-theme-background variable.

### What MDC Web Version are you using?

0.38.0

### What browser(s) is this bug affecting?

Chrome, Firefox, Edge

### What OS are you using?

Windows 10

### What are the steps to reproduce the bug?

```html
<div class="mdc-text-field mdc-text-field--textarea " data-mdc-auto-init="MDCTextField">
    <textarea class="mdc-text-field__input" type="text" rows="3" id="description"></textarea>
    <label class="mdc-floating-label" for="description">Description</label>
</div>
```

### What is the expected behavior?

The label's background color should be the same as the textarea's one.

### What is the actual behavior?

The label's background is just white:

```css
.mdc-text-field--textarea .mdc-floating-label {
    background-color: #fff;
    left: 1px;
    right: 0;
    margin-left: 8px;
    margin-right: 0;
    top: 18px;
    bottom: auto;
    margin-top: 2px;
    padding: 12px 8px 8px 8px;
    line-height: 1.15;
    pointer-events: none;
}
```

**Comment by acdvorak** [comment]
Thanks for filing this issue!

We're planning to update `textarea` to match the spec at some point (see #3231).

In the mean time, however, we should set the same background color on `textarea` and `label`.

**Comment by zandaqo** [comment]
This seems to be fixed in the latest release.

**Comment by kfranqueiro** [comment]
Thanks for pinging this issue. Yes, this was fixed by #4035 in the latest release because the textarea has been updated to use the notched outline component, so its label no longer has a background at all.
